### PR TITLE
Sparkbox Sprint 25

### PR DESCRIPTION
**Pull request by heatherdesigns**
# READY FOR REVIEW

## Summary

- feat(STN-542): Refine the Spotlight body text settings (#725)
- feat(STN-559): Hero Height Refinement (#721)
- feat(STN-527): Update External Link styles in Secondary Nav (#724)
- feat(STN-526): Refine Traditional theme Carousel height (#722)
- feat(STN-544): Add File and PDF icon links to Humsci Basic and legacy theme (#726) 
- feat(STN-543): Add spacing to buttons in a list on Node Detail pages (#727) 

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Ensure Code Climate passes against the `develop` branch. Link here: 
- [x] Assign to Mike for review.

## Need Review By (Date)
9/23

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
1. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

### CODE REVIEW

File: config/default/filter.format.minimal_html.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class> <span class> <table class> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class=""> <span class=""> <table class=""> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link">'
```
Comment (pookmish) [inline config/default/filter.format.minimal_html.yml]: This change is not necessary. it limits the classes that can be used on the elements. by specifying `class` alone, it allows any value for that attribute. if you provide `class="foo-bar"` the only class that is available is the `foo-bar` class. I'd rather allow any class. It also could have implications on the display of existing sites, including on the older themes.

**Review by pookmish — CHANGES_REQUESTED** [review]
Just a fixup on the one config, unless that was done intentionally for some specific reason.

File: config/default/filter.format.minimal_html.yml (hunk 1 above)
Comment (heatherdesigns) [reply config/default/filter.format.minimal_html.yml]: @pookmish I've added a commit to revert this change. :smile: Thanks!

Comment (pookmish) [reply config/default/filter.format.minimal_html.yml]: Thanks

**Review by pookmish — APPROVED** [review]

**Pull request merged**